eng/deployment/postgresql-kafka-opensearch/README.md

```diff
@@ -0,0 +1,70 @@
+# PostgreSQL -> Kafka -> OpenSearch
+
+> [!WARNING]
+> **NOT FOR PRODUCTION USE!** Includes passwords in the default configuration that are
+> visible in this repo and should never be used in real life. Be very careful!
+
+This document describes how to set up a development environment that uses Kafka and OpenSearch
+(along with PostgreSQL) to handle GET by queries. For requirements and design background, see
+[Change Data Capture to Stream](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/CDC-STREAMING.md).
+
+1. Start these containers.
+
+    ```shell
+    docker compose up -d
+    ```
+
+1. Open a shell on the PostgreSQL container and run the following to enable
+   replication:
+
+    ```bash
+    echo "host    replication    postgres         kafka-postgresql-source    trust" >> /var/lib/postgresql/data/pg_hba.conf
+    echo "wal_level = logical" >> /var/lib/postgresql/data/postgresql.conf
+    ```
+
+1. Restart the PostgreSQL container.
+1. Create the `edfi_datamanagementservice` database and run the DB installer. 
+1. Run the following SQL commands against the `edfi_datamanagementservice` database:
+    ```sql
+    CREATE PUBLICATION to_debezium WITH (publish = 'insert, update, delete, truncate', publish_via_partition_root = true);
+    SELECT pg_create_logical_replication_slot('debezium', 'pgoutput');
+    ALTER PUBLICATION to_debezium ADD TABLE dms.document;
+    ```
```

Comment:
I never took this step. Is this necessary because of the partitions?

Reply:
Yes, this is that fix. `publish_via_partition_root = true`

Reply:
@msilesgap here's another command that we need to run in the initial setup sidecar or script.